packages/style-engine/class-wp-style-engine.php

```diff
@@ -182,6 +191,12 @@ final class WP_Style_Engine {
 					'spacing' => '--wp--preset--spacing--$slug',
 				),
 			),
+			'width'       => array(
+				'property_keys' => array(
+					'default' => 'width',
+				),
+				'path'          => array( 'dimensions', 'width' ),
+			),
```

Comment:
This is just to support outputting `width` values for now — this PR doesn't propose adding a width block support just yet, but includes it in the structure of the `style` object for now so that it's a little tidier for outputting the `width: 100%` rule alongside the `aspect-ratio` rules.